apps/client/src/pages/Login/LoginComponents.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Heading,
+  useAuthenticator,
+  useTheme,
+  View,
+} from '@aws-amplify/ui-react';
+import { Button } from '@cloudscape-design/components';
+
+export const loginComponents = {
```

Comment:
Blocking: Could you add unit tests to assert the Amplify methods are called correctly? This will be easy if Amplify methods are passed in as props instead of directly depended on.